libs/pageserver_api/src/models/utilization.rs

```diff
@@ -104,6 +104,13 @@ impl PageserverUtilization {
         score >= 2 * Self::UTILIZATION_FULL
     }
 
+    /// The inverse of [`Self::score`].  Higher values are more affine to scheduling more work on this node.
+    ///
+    /// This uses the same threshold as [`Self::is_overloaded`] as a definition of "full".
+    pub fn free(&self) -> RawScore {
+        self.free_space_bytes
+    }
```

Comment:
Where is this used?

Reply:
Good catch, I was earlier playing with a different approach.  I've got way too accustomed to the `pub(crate)` safety net of the compiler telling me when things are unused.